backtrace-library/src/main/java/backtraceio/library/services/BacktraceHandlerInputReport.java

```diff
@@ -0,0 +1,41 @@
+package backtraceio.library.services;
+
+import backtraceio.library.events.OnServerErrorEventListener;
+import backtraceio.library.events.OnServerResponseEventListener;
+import backtraceio.library.events.RequestHandler;
+import backtraceio.library.models.BacktraceData;
+
+public class BacktraceHandlerInputReport extends BacktraceHandlerInput {
```

Comment:
Shouldn't it be `BacktraceReportHandlerInput` ?

Reply:
I was thinking it's a `BacktraceHandlerInput` of type `Report` so I put Report as a suffix. For metrics I have a `BacktraceHandlerInputEvents` which is `BacktraceHandlerInput` of type `Events`.

Reply:
Yes, sure!